src/applications/travel-pay/components/DecisionReason.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { STATUSES } from '../constants';
+
+const DecisionReason = ({ claimStatus, decisionLetterReason }) => (
+  <>
+    <p className="vads-u-font-weight--bold vads-u-margin-bottom--0">
+      {claimStatus === STATUSES.Denied.name
+        ? 'Why we denied your claim'
+        : 'Why we made a partial payment'}{' '}
```

Comment:
What is preventing the "Why we made a partial payment" text from appearing on a claim that was paid in full?

Reply:
sweet, thanks for the explanation